models/tokenlon/ethereum/tokenlon_v5_ethereum_amm_v1_trades.sql

```diff
@@ -46,6 +46,15 @@ WITH dexs AS (
     {% if is_incremental() %}
     WHERE evt_block_time >= date_trunc('day', now() - interval '7' day) 
     {% endif %}
+), prices AS (
+    SELECT DISTINCT
+      DATE_TRUNC('hour', minute) AS hour,
+      contract_address,
+      blockchain,
+      decimals,
+      AVG(price) AS price
+    FROM "delta_prod"."prices"."usd"
```

Comment:
We may not have the hardcoding source here.
```suggestion
    FROM {{ source('prices', 'usd') }}
```